l10n_es/data/tax_codes_common.xml

```diff
@@ -1073,6 +1085,12 @@
       <field name="name">IRPF retenciones a cuenta. Base imponible (19%)</field>
       <field name="sign">1.0</field>
     </record>
+    <record id="account_tax_code_template_IRACBI195" model="account.tax.code.template">
```

Comment:
Para las retenciones a cuenta de arrendamientos no se ponen códigos de impuestos (no se declaran en ningún modelo periódico, si no en el IS - modelo 200, pero se hace con saldo de la cuenta 473), pero ya lo corrijo yo al hacer merge.